opentok-angular.js

```diff
@@ -147,6 +147,7 @@ ng.module('opentok', [])
             props.videoContentHint = 'detail';
             publisherVideo = document.createElement('video');
             navigator.mediaDevices.getDisplayMedia({ video: { width, height }, audio: false }).then((stream) => {
+              scope.streamForCanvas = stream.getTracks()
```

Comment:
`streamForCanvas` doesn't work for me because these are tracks.
I guess `canvasStreamTracks` would be my suggestions

Reply:
sure that works for me